lib/replaceRef.js

```diff
@@ -6,38 +6,40 @@ var fs = require('fs');
 
 module.exports = replaceRef;
 
-// TODO: allow this to be command-line configurable
-var ignorableTypes = ['gltfid.schema.json', 'gltfchildofrootproperty.schema.json', 'gltfproperty.schema.json'];
```

Comment:
Also the help doesn't really explain how to pass a correct parameter list to `-i`.  I had to read the source code and try a couple times to get it.